CONTRIBUTING.md

```diff
@@ -277,7 +277,7 @@ As discussed in [*How we're organized*](#how-were-organized), this role only req
 
 ### How to adopt pull requests
 
-Occasionally authors of pull requests get busy or become unresponsive.
+Occasionally authors of pull requests get busy or become unresponsive, in other hand, on the project side, sometimes members fail to reply in a timely manner.
```

Comment:
```suggestion
Occasionally authors of pull requests get busy or become unresponsive, or project members fail to reply in a timely manner.
```

Reply:
Just cleaning up the wording a bit then this LGTM!

Reply:
Done! Thanks for the suggestion.